src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,63 @@
+package duke.task;
+
+/**
+ * Task class which encapsulates task description and status of completion.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor of Task class.
+     *
+     * @param description Description of task.
+     */
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    public boolean getIsDone() {
+        return isDone;
+    }
+
+    /**
+     * Returns "X" if uncompleted and " " otherwise.
```

Comment:
Should this line be "Returns "X" if completed and " " otherwise."?

Reply:
Wow, that's very observant. Thanks for pointing that out. 💯